templates/server.properties.j2

```diff
@@ -1,27 +1,21 @@
 ##
 # {{ ansible_managed }}
 
-##
-# Server Basics
+############################# Server Basics #############################
```

Comment:
I really don't mind either way to be honest, I can update the other headers if you want me to?